### DISCUSSION TITLE
HELP. I don't know how to give Monika the dress I drew.

### DISCUSSION THREAD

**Discussion by howltrek** [Monika-After-Story/MonikaModDev#8029] [QA - code and content creation (technical)]
I drew this suit for her last night, but no matter how hard I have tried, I can't create the files well to give it to her. Please Help

![Black_Dress_whit_rose](https://user-images.githubusercontent.com/90975566/136797196-70a551f5-dcb9-4fcf-bb8c-456cd4c5bbe0.png)
![body-def-0](https://user-images.githubusercontent.com/90975566/136797202-b511a3be-832e-4304-ad60-2febbbe6b187.png)
![body-def-1](https://user-images.githubusercontent.com/90975566/136797206-c1f70b6a-ab0c-4f7a-8ff1-76a735e6368a.png)
![body-leaning-def-0](https://user-images.githubusercontent.com/90975566/136797207-68715714-d0ab-4c15-ac17-3b21f28a8d2c.png)
![body-leaning-def-1](https://user-images.githubusercontent.com/90975566/136797208-4a412e3c-034d-4f50-a32b-a769b0e3401d.png)

**Comment by Booplicate** [comment]
Have you already seen [this guide](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#creating-sprite-packs)?

**Reply by howltrek** [reply]
thacks I try but, fail in some thig :/

**Reply by Booplicate** [reply]
You need to be more specific in what exactly doesn't work. Do you get an error?

**Reply by howltrek** [reply]
Thank you, I already corrected the problem and she recognized the dress well
